nautobot/utilities/testing/api.py

```diff
@@ -318,6 +320,11 @@ def test_create_object(self):
                     exclude=self.validation_excluded_fields,
                     api=True,
                 )
+                # Check if Slug field is automatically created
```

Comment:
Does it make sense to add a similar check to `test_bulk_create_objects` below?

Reply:
Sure, why not.